akka-cluster-sharding/src/main/scala/akka/cluster/sharding/internal/EventSourcedRememberEntities.scala

```diff
@@ -97,23 +97,23 @@ private[akka] final class EventSourcedRememberEntitiesStore(
   override def snapshotPluginId: String = settings.snapshotPluginId
 
   override def receiveRecover: Receive = {
-    case EntityStarted(id)                 => state = state.copy(state.entities + id)
+    case EntitiesStarted(ids)              => state = state.copy(state.entities ++ ids)
```

Comment:
was there something about `Set.union` being much faster?

Reply:
It's impl is def union(that: GenSet[A]): This = this ++ that